apps/admin/src/app/dashboard/tracks/[trackId]/_actions/update-track.action.ts

```diff
@@ -20,7 +20,7 @@ export async function updateTrack(data: FormSchema & { trackId: number }) {
       trackChallenges: {
         createMany: {
           data: trackChallenges.map((tc) => {
-            const { challenge, ...rest } = tc;
+            const { challenge: _, ...rest } = tc;
```

Comment:
[Explanation] Another unused variable per [`@typescript-eslint/no-unused-vars`](https://typescript-eslint.io/rules/no-unused-vars). It can't be removed altogether, though, because then it'd be included in `...rest`. So `_` is used to discard it.

Fun fact, there's a TC39 proposal in the works to make `void` a more standard name for these "discard bindings". https://github.com/tc39/proposal-discard-binding ✨